### PR TITLE
Switch from mapbox to static SVG map

### PR DESCRIPTION
**Pull request by liquidx**
@reustle 

I replaced the MapBox map with a static SVG map with prefecture level highlighting. Whatcha think?

Screen Shot 2020-02-27 at 22 54 24

The map is licensed CC-BY-NC.

### CODE REVIEW

File: static/main.js [hunk 1]
```diff
@@ -276,68 +244,38 @@ function drawPageTitleCount(confirmed) {
...
-  })
 
-  // Start the Mapbox search expression
-  let prefecturePaint = [
-    'match',
-    ['get', 'NAME_1'],
-  ]
-  
   // Go through all prefectures looking for cases
+  let japanMapSvg = document.querySelector(`#map-japan-svg`).contentDocument
```
Comment (reustle) [inline static/main.js]: \` isn't supported in IE unfortunately

**Comment by python273** [comment]
I'd also crop svg, so it's zoomed in a bit more

**Comment by liquidx** [comment]
Let me take a look and try this evening to make it work across mobile and IE.

Zooming should be doable, I think I should add some labels which is also possible.

File: static/main.js (hunk 1 above)
Comment (liquidx) [reply static/main.js]: Fixed

**Comment by liquidx** [comment]
Fixed layout on mobile, but didn't add zooming.

Looks like the site is still broken on IE somewhat, even before getting to drawing the map.

**Comment by olingern** [comment]
ping @reustle, looks like this is waiting for review / comment

**Comment by reustle** [comment]
Sorry, @liquidx and I talk offline a lot. Currently wrangling how mobile will work, and how we would then move to city-based data instead of prefecture-based. Kind of 'on-hold'.

**Comment by reustle** [comment]
Do we want to shelve this PR for the time being?

**Comment by olingern** [comment]
I think so. Will close :+1:

**Comment by liquidx** [comment]
Closing this PR. HEAD has deviated so far and we ended up being OK with the MapBox map that I think it's fine to leave it as is for now.

**Pull request closed without merging**